src/floris/tools/visualization.py

```diff
@@ -23,7 +23,7 @@
 from matplotlib import rcParams
 
 
-def plot_turbines(ax, layout_x, layout_y, yaw_angles, D, color=None, wind_direction=270.0):
+def plot_turbines(ax, layout_x, layout_y, yaw_angles, rotor_diameters, color=None, wind_direction=270.0):
```

Comment:
There was a comment later in the code about "grabbing diameter" except it was D and so obviously the remnants of my 16 year old self took over. So I thought we ought to avoid a future awkward situation and start calling diameter by its right name.